lib/pip.js

```diff
@@ -50,29 +51,39 @@ function installRequirements() {
 
     this.serverless.cli.log(`Docker Image: ${this.options.dockerImage}`);
 
-    // Check docker server os type from 'docker version'
-    let volPath;
+    // Determine os platform of docker CLI from 'docker version'
     options = [
-      'version', '--format', '{{with .Server}}{{.Os}}{{end}}'
+      'version', '--format', '{{with .Client}}{{.Os}}{{end}}'
     ];
-    const ps = spawnSync(cmd, options, {'timeout': 2000, 'encoding': 'utf-8'});
+    const ps = spawnSync(cmd, options, {'timeout': 10000, 'encoding': 'utf-8'});
```

Comment:
Changed timeout to 10 seconds just in case some automated build process have a hard time contacting its docker server/backend engine.